src/main/java/uk/gov/hmcts/dts/fact/services/admin/AdminService.java

```diff
@@ -106,4 +106,9 @@ public Court save(String slug, Court court) {
     public void updateMultipleCourtsInfo(CourtInfoUpdate info) {
         courtRepository.updateInfoForSlugs(info.getCourts(), info.getInfo(), info.getInfoCy());
     }
+
+    @Transactional
+    public void updateCourtLatLon(final String slug, final Double lat, final Double lon) {
+        courtRepository.updateLatLonBySlug(slug, lat, lon);
```

Comment:
Double check based on the comment I left above, but you may be able to set the values of lat and long here for the court entity, and then save it to do an update...but needs double checking, and I could be wrong. 

From what I've read there is some weird under the hook magic around what save() does based upon id's and such, and persistance states